lib/acts_as_indexed/class_methods.rb

```diff
@@ -134,25 +137,59 @@ def search_index(query, find_options={}, options={})
 
       return part_query if options[:ids_only]
 
-      with_scope :find => find_options do
-        # Doing the find like this eliminates the possibility of errors occuring
-        # on either missing records (out-of-sync) or an empty results array.
-        records = find(:all, :conditions => [ "#{table_name}.#{primary_key} IN (?)", part_query])
-
-        if find_options.include?(:order)
-         records # Just return the records without ranking them.
+      # New 2023 way to find the records.
+      conditions = ["#{table_name}.#{primary_key} IN (?)", part_query]
+      query_options = find_options.slice(:limit, :offset, :order)
```

Comment:
It looks like this is assigned but never used?

Reply:
Oh yeah, this was part of me working this out, no longer necessary, removing it.